### PR TITLE
Milestone 2:Basic components

### PR DESCRIPTION
**Pull request by AdnanAfsari**

### CODE REVIEW

**Comment by AdnanAfsari** [comment]
Stickler errors will be solved in next milestone when i will merge my milestone 1 so please ignore it for this milestone 😄

File: src/components/App/App.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './App.css';
+import Display from '../Display/Display';
+import ButtonPanel from '../ButtonPanel/ButtonPanel'
+
+class App extends React.Component {
+  render() {
+    return (
+      <div className="App">
+          <Display />
```
Comment (rvvergara) [inline src/components/App/App.js]: `Display` is supposed to receive a props called `value`. For now we can assign this `value` prop to any string. The point is we just want `Display` to render correctly and for us to verify it does. Later on we will revise this prop to something meaningful

File: src/components/ButtonPanel/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Button from '../Button/Button';
+
+const buttonPanel = () => {
+  return (
+    <div>
+      <div ClassName = "group-1">
```
Comment (rvvergara) [inline src/components/ButtonPanel/ButtonPanel.js]: Typo error `ClassName`, causes an error in the console. It should be `className`

**Comment by AdnanAfsari** [comment]
I made all the changes you requested 😄 , Thanks

**Pull request merged**